graylog2-server/src/main/java/org/graylog2/bindings/providers/EsNodeProvider.java

```diff
@@ -48,43 +47,45 @@ public EsNodeProvider(ElasticsearchConfiguration configuration) {
     }
 
     @Override
-    @Singleton
```

Comment:
Aye, I just tested it.

- `@Singleton` on the `get()` method: does nothing
- `@Singleton` on the class: caches the provider instance, not the object returned from `get()`
- `.in(Scopes.SINGLETON)` or `.asEagerSingleton()` in bindinds: caches both, `get()` method return *and* the provider instance